src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer starts work");

```

Comment:
The message 'Bulldozer starts work' could be more informative by specifying that it's a Bulldozer machine that is working, to adhere to the checklist requirement.